extension/test/data_test.ts

```diff
@@ -86,6 +86,12 @@ describe('data.ts', function () {
   });
 
   describe('makeText', function () {
+    it('returns an empty string if null dict entry is passed in', function () {
+      const emptyString = rcxDict.makeText(null, /* max= */ 1);
```

Comment:
(super nit) It's a bit strange to call this `emptyString` already since that's what we're asserting later. Usually, this would just be something like `result` or `text`. By embedding the expecation in the actual value it might lead readers to not read the assertion line as closely leading to eventual drift.

I actually realize now this is similar to what you've done in the other tests and I wouldn't change it in this PR.

Reply:
Yeah, that makes sense. I can change all the variable names so they aren't like this.